game/ui/components/agentassignment.cpp

```diff
@@ -694,6 +700,7 @@ void AgentAssignment::eventOccured(Event *e)
 			// if dragged - change location of dragged list
 			if (isDragged)
 			{
+				isMouseMoved = true;
```

Comment:
Modern mice have too sensitive sensors. Often when user do a click the cursor may shifts to several pixels. It's especially issue when need to check a difference between drag&drop and a conventional click especially if the MouseDown and MouseUp events handled separately. Thus added insensibility area (5x5 pixels in this case).
You set isMouseMoved=true just after MouseMove event. In this case if user want to load the equip screen his mouse must not be shifted. Or the user should click again.

May be better to relax a demands to users precision and allow to click in the insensibility area? Moreover, almost all code done below.

Reply:
It works without problems with my (not cheap) high precision Razor mouse.

Reply:
Just now tested. If mouse moved even one pixel the equip screen did not start.
The vanilla X-Com works well even if mouse shifted a bit during the click.